models/dex/dex_offers.sql

```diff
@@ -0,0 +1,53 @@
+{{ config(
+        alias ='offers',
+        post_hook='{{ expose_spells(\'["ethereum", "bnb", "avalanche_c", "gnosis", "optimism", "arbitrum", "fantom", "polygon"]\',
```

Comment:
For now we only keep this for optimism chain.
```suggestion
        post_hook='{{ expose_spells(\'["optimism"]\',
```